### PR TITLE
Added percentage symbol to BP header and values in Dashboard

### PR DESCRIPTION
**Pull request by timmyjose**
Story: https://www.pivotaltracker.com/story/show/165672235

### CODE REVIEW

File: app/helpers/dashboard_helper.rb [hunk 1]
```diff
@@ -4,7 +4,11 @@ def lighten_if_zero(value)
   end
 
   def dash_if_zero(value)
-    (value == nil || value == 0) ? "-" : value
+    is_zero(value) ? "-" : value
+  end
+
+  def is_zero(value)
+    value == nil || value == 0
```
Comment (kitallis) [inline app/helpers/dashboard_helper.rb]: Potentially,
```suggestion
    value.nil? || value.zero?
```

File: app/helpers/dashboard_helper.rb [hunk 2]
```diff
@@ -4,7 +4,11 @@ def lighten_if_zero(value)
   end
 
   def dash_if_zero(value)
-    (value == nil || value == 0) ? "-" : value
+    is_zero(value) ? "-" : value
+  end
+
+  def is_zero(value)
```
Comment (kitallis) [inline app/helpers/dashboard_helper.rb]:
```suggestion
  def zero?(value)
```

Comment (timmyjose) [reply app/helpers/dashboard_helper.rb]: Updated the method name to be more idiomatic.

File: app/helpers/dashboard_helper.rb (hunk 1 above)
Comment (timmyjose) [reply app/helpers/dashboard_helper.rb]: Updated to use methods.

**Pull request merged**